src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,15 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  filter: string
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, filter }) => {
+  const [copyList, setCopyList] = useState(([...movies]));
+
+  useEffect(() => {
+    if (filter) {
+      setCopyList(movies.filter((el: Movie) => {
+        if (
+          el.title.toLocaleLowerCase()
+            .includes(filter)
```

Comment:
```suggestion
            .includes(filter.trim().toLowerCase())
```

Reply:
I don't agree, it's easier to trim the space and reduce the text here, in one place than in another file in two

Reply:
yep, I agree with you, good job 👍